taxi/views.py

```diff
@@ -64,14 +65,14 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
     success_url = reverse_lazy("taxi:car-list")
+    form_class = CarForms
 
 
 class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
     model = Car
-    fields = "__all__"
     success_url = reverse_lazy("taxi:car-list")
+    form_class = CarForms
```

Comment:
The `CarUpdateView` also uses `CarForms` as its form class. Verify that `CarForms` is suitable for updating car instances, including handling any specific fields or validation required for updates.